2021-03-25 15:06:33,789 (28307/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-03-25 15:06:33,789 (28307/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-03-25 15:06:33,792 (28307/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/opt/homebrew/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/opt/homebrew/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\npsycopg2.OperationalError: could not connect to server: No such file or directory\n\tIs the server running locally and accepting\n\tconnections on Unix domain socket "/tmp/.s.PGSQL.5432"?\n\n'
